lightwood/analysis/nc/nc.py

```diff
@@ -327,23 +327,23 @@ def score(self, x, y=None):
             Nonconformity scores of samples.
         """ # noqa
         prediction = self.model.predict(x)
-        n_test = x.shape[0]
 
-        norm = np.ones(n_test)
+        err = self.err_func.apply(prediction, y)
         if self.normalizer is not None:
             try:
                 norm = self.normalizer.score(x) + self.beta
+                err = err / norm
             except Exception:
                 pass
 
-        return self.err_func.apply(prediction, y) / norm
```

Comment:
prior use case err / norm is passed and self.normalizer changes the norm. Is this intended?

Reply:
see my other comment, error still gets normed but we streamline the case where `norm=1`